### PR TITLE
Add "Unit-aware operations (Pint)" training notebook

### LINKED ISSUE #32: Create a "MetPy (unit-aware calculations)" Training Notebook
Following the [instructors/notebook-template/Template.ipynb](https://github.com/Unidata/pyaos-ams-2021/blob/master/instructors/notebook-template/Template.ipynb) template notebook, create a training notebook that lives in `notebooks/analylsis/` directory that covers the ability of MetPy to perform unite-aware calculations to assist workshop participants who choose to work on the following projects:
  * [`site/projects/isentropic_analysis.md`](https://unidata.github.io/pyaos-ams-2021/projects/isentropic_analysis.html)
  * [`site/projects/severe_wx_outbreak.md`](https://unidata.github.io/pyaos-ams-2021/projects/severe_wx_outbreak.html)

Once the notebook is complete, be sure to do the following:

* [ ] clear the output of the notebook
  * jupyter-notebook interface: `Cell` &rarr; `All Output` &rarr; `Clear`
  * jupyter-lab interface: `Right-click` &rarr; `Clear All Outputs`
* [ ] link the notebook in the following locations using the a nbviewer (`https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/analysis/<training-notebook-name>.ipynb`):
  * [ ] the project page for each project idea listed above
  * [ ] the project entry in [`site/resources.md`](https://unidata.github.io/pyaos-ams-2021/resources.html)
* [ ] add a description of the notebook to project entry in [`site/resources.md`](https://unidata.github.io/pyaos-ams-2021/resources.html)

### PR DESCRIPTION
**Pull request by rpmanser**
Closes #32

### CODE REVIEW

File: notebooks/analysis/units.ipynb [hunk 1]
```diff
@@ -0,0 +1,575 @@
...
+    "* Introduce the use of `units` to track values with physical units\n",
+    "* Demonstrate basic operations with and on values with units\n",
+    "* Show examples of using units in `MetPy`\n",
+    "\n",
+    "\n",
+    "### Objectives\n",
+    "1. [Introduction to Units](#1.-Introduction-to-Units)\n",
+    "1. [Quantity Methods and Attributes](#2.-Quantity-Methods-and-Attributes)\n",
+    "1. [MetPy and Units](#3.-MetPy-and-Units)\n",
+    "1. [Gotchas](#4.-Gotchas-)\n",
```
Comment (srcarter3) [inline notebooks/analysis/units.ipynb:36]: This link is broken, I believe removing the trailing '-' should fix it.

File: notebooks/analysis/units.ipynb [hunk 2]
```diff
@@ -0,0 +1,575 @@
...
+   "metadata": {},
+   "source": [
+    "## 1. Introduction to Units\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`MetPy` provides units support through the `Pint` library. Units in Python are, as described by the [`Pint` documentation](https://pint.readthedocs.io/en/stable/), \"the product of a numerical value and a unit of measurement.\" They are used to programatically keep track of physical quantities so we can worry less about unit conversion errors and focus on our coding objective instead.\n",
```
Comment (srcarter3) [inline notebooks/analysis/units.ipynb:76]: Is this supposed to say "Units in Python..." or "Values in Python..."?

File: site/resources.md [hunk 3]
```diff
@@ -75,8 +75,8 @@ The end result is a tighter vertical spacing between the opening line of the bul
   _add description_
 
 * Unit-aware operations (Pint)
-
-  _add description_
+* [**Unit-aware operations (Pint)**](https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/analysis/units.ipynb)
```
Comment (srcarter3) [inline site/resources.md:78]: Title line needs two spaces at the end (see comment in top of file as well as existing entries for reference).

**Review by srcarter3 — CHANGES_REQUESTED** [review]
Minor markdown and notebook changes requested.
Looks good, and I enjoyed the notebook!

File: notebooks/analysis/units.ipynb (hunk 2 above)
Comment (rpmanser) [reply notebooks/analysis/units.ipynb:76]: Not quite either, I just worded it poorly. Fix incoming shortly.

**Pull request merged**